webapp/src/components/modals/actionModal.tsx

```diff
@@ -0,0 +1,173 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Table from 'react-bootstrap/Table';
+import Form from 'react-bootstrap/Form';
+
+import './styles.css';
+
+import {OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+type Props = {
+    visibility: boolean;
+    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
+    config: Configs[];
+    configIndex: number;
+}
+
+const ActionModal = ({visibility, setVisibility, config, configIndex}: Props) => {
+    const [show, setShow] = useState(false);
+
+    const [attachmentMessageAvailable, setAttachmentMessageAvailable] = useState(false);
+
+    const actionsLength = config[configIndex]?.actions?.length ?? 0;
```

Comment:
```suggestion
    const actionsLength = config[configIndex]?.actions?.length;
```
I think this will work!